fish/install.sh

```diff
@@ -2,6 +2,14 @@
 set -e
 set -u
 
+if command -v fish > /dev/null; then
+    if [ ! -e ~/.config/fish/config.fish ]; then
```

Comment:
Why not wrap this in a function if its gonna call the exact same code twice?

Reply:
My general rule is that if something is relatively small, I'll write it twice. If I write it 3 times, then I'll consider an abstraction.